
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>{{ name }} - TDC</title>
    <link rel="stylesheet" href="{{ url_for('static', filename='index.css') }}">
    <link rel="shortcut icon" type="image/png"
           href="{{ url_for('static', filename='images/favicon.png') }}" 
    />
    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
    <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>{{ name }} | TDC</title>
<meta name="generator" content="Jekyll v3.9.3" />
<meta property="og:title" content="{{ name }}" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Artificial intelligence foundation for therapeutic science" />
<meta property="og:description" content="Artificial intelligence foundation for therapeutic science" />
<link rel="canonical" href="http://localhost:4000/single_pred_tasks/tox" />
<meta property="og:url" content="http://localhost:4000/single_pred_tasks/tox" />
<meta property="og:site_name" content="TDC" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="{{ name }}" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","description":"Artificial intelligence foundation for therapeutic science","headline":"Toxicity","url":"http://localhost:4000/single_pred_tasks/tox"}</script>
<!-- End Jekyll SEO tag -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-183522810-1"></script>
<script>
  window['ga-disable-UA-183522810-1'] = window.doNotTrack === "1" || navigator.doNotTrack === "1" || navigator.doNotTrack === "yes" || navigator.msDoNotTrack === "1";
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-183522810-1');
</script><!-- head scripts --></head>

  <body>
    
<nav class="navbar is-primary" >
    <div class="container">
        <div class="navbar-brand">
            <a class="navbar-brand" href="/">
            <span><img src="{{ url_for('static', filename='images/logonav.png') }}" alt="Logo" style="height: auto; width: auto; max-height: 45px; max-width: 250px;"></span>
            </a>
            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
            </a>
        </div>
        <div class="navbar-menu" id="navMenu">
            <div class="navbar-end">
                
                
                    
                    <a href="/" class="navbar-item ">Home</a>
                    
                
                    
                    <a href="/start/" class="navbar-item ">Start</a>
                    
                
                    
                    <div class="navbar-item has-dropdown is-hoverable">
                        <a href="/overview" class="navbar-link ">Datasets</a>
                        <div class="navbar-dropdown">
                            
                            <a href="/overview" class="navbar-item ">Overview</a>
                            
                            <a href="/single_pred_tasks/overview" class="navbar-item ">Single-instance Prediction</a>
                            
                            <a href="/multi_pred_tasks/overview" class="navbar-item ">Multi-instance Prediction</a>
                            
                            <a href="/generation_tasks/overview" class="navbar-item ">Generation</a>
                            
                        </div>
                    </div>
                    
                
                    
                    <div class="navbar-item has-dropdown is-hoverable">
                        <a href="/fct_overview" class="navbar-link ">Data Functions</a>
                        <div class="navbar-dropdown">
                            
                            <a href="/fct_overview" class="navbar-item ">Overview</a>
                            
                            <a href="/functions/data_evaluation" class="navbar-item ">Model Evaluation</a>
                            
                            <a href="/functions/data_split" class="navbar-item ">Dataset Splits</a>
                            
                            <a href="/functions/data_process" class="navbar-item ">Data Processing</a>
                            
                            <a href="/functions/oracles" class="navbar-item ">Molecule Generation Oracles</a>
                            
                        </div>
                    </div>
                    
                
                    
                    <div class="navbar-item has-dropdown is-hoverable">
                        <a href="/benchmark/overview" class="navbar-link ">Leaderboards</a>
                        <div class="navbar-dropdown">
                            
                            <a href="/benchmark/overview" class="navbar-item ">Guidelines</a>
                            
                            <a href="/benchmark/admet_group/overview" class="navbar-item ">ADMET Group</a>
                            
                            <a href="/benchmark/drugcombo_group/overview" class="navbar-item ">DrugCombo Group</a>
                            
                            <a href="/benchmark/docking_group/overview" class="navbar-item ">Docking Group</a>
                            
                            <a href="/benchmark/dti_dg_group/overview" class="navbar-item ">DTI DG Group</a>
                            
                            <a href="/benchmark/scdti_group/overview" class="navbar-item ">Single-cell DTI Group</a>
                            
                            <a href="/benchmark/proteinpeptide_group/overview" class="navbar-item ">Protein-Peptide Binding Affinity Group</a>
                            
                            <a href="/benchmark/counterfactual_group/overview" class="navbar-item ">Counterfactual Prediction Group</a>
                            
                            <a href="/benchmark/clinical_trial/overview" class="navbar-item ">Clinical Trial Outcome Prediction Group</a>
                            
                        </div>
                    </div>
                    
                
                    
                    <a href="https://huggingface.co/tdc" class="navbar-item ">HF Models</a>
                    
                
                    
                    <a href="/news" class="navbar-item ">News</a>
                    
                
                    
                    <a href="/team" class="navbar-item ">Team</a>
                    
                
                <a href="https://zitniklab.hms.harvard.edu/publications/papers/TDC-neurips21-main.pdf" class="navbar-item">NeurIPS Paper</a>
                <a href="https://www.nature.com/articles/s41589-022-01131-2" class="navbar-item">Nat Chem Bio Paper</a>
                <a href="https://tdc.readthedocs.io" class="navbar-item">Docs</a>
                <a href="https://github.com/mims-harvard/TDC" class="navbar-item">GitHub</a>
                
            </div>

        </div>
    </div>
</nav>

    
    


    <section class="section">
        <div class="container">
            <div class="columns">
                
                <div class="column is-4-desktop is-4-tablet">
                    

<aside class="menu">

    <p class="menu-label"></p>
    <ul class="menu-list">
        
        <li>
            <a href="/single_pred_tasks/overview" class=""><strong>Single-instance Prediction Problem</strong></a>
            
            <ul>
                
                {% if task == "adme" %} 
                <li><a href="/single_pred_tasks/adme" class="is-active"><code>ADME</code> Pharmaco-kinetics</a></li>
                {% else %}
                <li><a href="/single_pred_tasks/adme" class=""><code>ADME</code> Pharmaco-kinetics</a></li>
                {% endif %}
                
                
                {% if task == "tox" %} 
                <li><a href="/single_pred_tasks/tox" class="is-active"><code>Tox</code> Toxicity</a></li>
                {% else %}
                <li><a href="/single_pred_tasks/tox" class=""><code>Tox</code> Toxicity</a></li>
                {% endif %}
                
                
                {% if task == "hts" %}
                <li><a href="/single_pred_tasks/hts" class="is-active"><code>HTS</code> High-Throughput Screening</a></li>
                {% else %}
                <li><a href="/single_pred_tasks/hts" class=""><code>HTS</code> High-Throughput Screening</a></li>
                {% endif %}
                
                {% if task == "qm" %}
                <li><a href="/single_pred_tasks/qm" class="is-active"><code>QM</code> Quantum Mechanics</a></li>
                {% else %}
                <li><a href="/single_pred_tasks/qm" class=""><code>QM</code> Quantum Mechanics</a></li>
                {% endif %}
                
                
                {% if task == "yields" %}
                <li><a href="/single_pred_tasks/yields" class="is-active"><code>Yields</code> Reaction Yields Prediction</a></li>
                {% else %}
                <li><a href="/single_pred_tasks/yields" class=""><code>Yields</code> Reaction Yields Prediction</a></li>
                {% endif %}
                
                
                {% if task == "epitope" %}
                <li><a href="/single_pred_tasks/epitope" class="is-active"><code>Epitope</code> Epitope Prediction</a></li>
                {% else %}
                <li><a href="/single_pred_tasks/epitope" class=""><code>Epitope</code> Epitope Prediction</a></li>
                {% endif %}
                
                
                {% if task == "develop" %}
                <li><a href="/single_pred_tasks/develop" class="is-active"><code>Develop</code> Developability Prediction</a></li>
                {% else %}
                <li><a href="/single_pred_tasks/develop" class=""><code>Develop</code> Developability Prediction</a></li>
                {% endif %}
                
                
                {% if task == "CRISPROutcome" %}
                <li><a href="/single_pred_tasks/CRISPROutcome" class="is-active"><code>CRISPROutcome</code> CRISPR Repair Prediction</a></li> 
                {% else %}
                <li><a href="/single_pred_tasks/CRISPROutcome" class=""><code>CRISPROutcome</code> CRISPR Repair Prediction</a></li>
                {% endif %}
                
                
            </ul>
            
        </li>
            
    </ul>

    <p class="menu-label"></p>
    <ul class="menu-list">
        
        <li>
            <a href="/overview" class=""><strong>Others</strong></a>
            
            <ul>
                
                
                <li><a href="/multi_pred_tasks/overview" class="">Multi-instance Prediction Problem</a></li>
                
                
                
                <li><a href="/generation_tasks/overview" class="">Generation Problem</a></li>
                
                
            </ul>
            
        </li>
            
    </ul>

</aside>
                </div>
                
                <div class="column is-8">
                    
                    
                    
                    
    
    

<div class="contents">
    <div class="menu">
        <p class="menu-label">Dataset Index</p>
        <ul class="menu-list">
    {% for var, id, name, desc, task, stats, splits, class, refs, license, liclink in datasets %}
    <li><a href="#{{ id }}">{{ name }}</a></li>
    {% endfor %}
    {% if task == "tox" %}
    <li><a href="#herg-central">hERG Central</a></li>
    <li><a href="#tox21">Tox21</a></li>
    <li><a href="#toxcast">ToxCast</a></li>
    {% endif %}
    {% if task == "hts" %}
    <li><a href="#butkiewicz-et-al">Butkiewicz et al.</a></li>
    {% endif %}
    {% if task == "qm" %}
    <li><a href="#qm7b">QM7b</a></li>
    <li><a href="#qm8">QM8</a></li>
    <li><a href="#qm9">QM9</a></li>
    {% endif %}
    {% if task == "develop" %}
    <li><a href="#tap">TAP</a></li>
    <li><a href="#sabdab-chen-et-al">SAbDab, Chen et al.</a></li>
    {% endif %}
    {% if task == "CRISPROutcome" %}
    <li><a href="#leenay-et-al">Leenay et al.</a></li>
    {% endif %}
</ul>
    </div>
</div>




<div class="content">
    <h1 id="{{ desc_id }}">{{ desc_title }}</h1>

<div class="box">
	

<p class="is-size-6">  <strong> Definition: </strong> 
{{ desc_desc }}
</p>

<p class="is-size-6"> <strong> Impact: </strong>  
{{ desc_impact }} 
</p>

<p class="is-size-6"> <strong> Generalization: </strong>  {{ desc_gen }}  </p>

<p class="is-size-6"> <strong> Product: </strong> {{ desc_product }} </p>

<p class="is-size-6"> <strong> Pipeline: </strong>{{ desc_pipeline }} </p>

</div>

{% for var, id, name, desc, task, stats, splits, class, refs, license, liclink in datasets %}
<h3 id="{{ id }}">{{ name }}</h3>
<p class="is-size-6">  <strong> Dataset Description: </strong> {{ desc }} </p>
<p class="is-size-6">  <strong> Task Description: </strong>{{ task }}</p>
<p class="is-size-6">  <strong> Dataset Statistics: </strong>{{ stats }}</p>
<p class="is-size-6">  <strong> Dataset Split: </strong> 
{% for split in splits %}
<span class="tag is-info is-light">{{split}}</span>
{% endfor %}
</p>
<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
<span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">{{ class }}</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">{{ class }}</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'{{ var }}'</span><span class="p">)</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>
<p class="is-size-6">  <strong> References: </strong>  </p>
{% for ref, reflink in refs %}
<p><a href="{{ reflink }}">{{ ref }}</a></p>
{% endfor %}
<p class="is-size-6"> <strong> Dataset License: </strong> <a href="{{ liclink }}">{{ license }}</a></p>
<hr />
{% endfor %}

{% if task == "tox" %}
<h3 id="herg-central">hERG Central</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong>  Human ether-à-go-go related gene (hERG) is crucial for the coordination of the heart's beating. Thus, if a drug blocks the hERG, it could lead to severe adverse effects. Therefore, reliable prediction of hERG liability in the early stages of drug design is quite important to reduce the risk of cardiotoxicity-related attritions in the later development stages. There are three targets: hERG_at_1uM, hERG_at_10uM, and hERG_inhib. </p>

<p class="is-size-6">  <strong> Task Description: </strong> </p>

<ul>
  <li>hERG_at_1uM: Regression. Given a drug SMILES string, predict the percent inhibition at a 1µM concentration.</li>
  <li>hERG_at_10uM: Regression. Given a drug SMILES string, predict the percent inhibition at a 10µM concentration.</li>
  <li>hERG_inhib: Binary classification. Given a drug SMILES string, predict whether it blocks (1) or not blocks (0). This is equivalent to whether hERG_at_10uM &lt; -50, i.e. whether the compound has an IC50 of less than 10µM.</li>
</ul>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> 306,893 drugs. </p>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split</span> <span class="tag is-info is-light">Scaffold Split</span>  </p>

<p class="is-size-6">  <strong> Note: </strong> Tox21 contains multiple assays data. To retrieve the specific labels for that assay, specify the label name in the <code>label_name</code> variable to the data loader. You can find all available label names by calling: </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.utils</span> <span class="kn">import</span> <span class="n">retrieve_label_name_list</span>
<span class="n">label_list</span> <span class="o">=</span> <span class="n">retrieve_label_name_list</span><span class="p">(</span><span class="s">'herg_central'</span><span class="p">)</span>
</code></pre></div></div>
<p class="is-size-6"> Then, go to the standard TDC data loader procedure with the label name specified. </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">Tox</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">Tox</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'herg_central'</span><span class="p">,</span> <span class="n">label_name</span> <span class="o">=</span> <span class="n">label_list</span><span class="p">[</span><span class="mi">0</span><span class="p">])</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6">  <strong> References: </strong>  </p>

<p><a href="https://pubmed.ncbi.nlm.nih.gov/22149888/">[1] Du F, Yu H, Zou B, Babcock J, Long S, Li M. hERGCentral: a large database to store, retrieve, and analyze compound-human Ether-à-go-go related gene channel interactions to facilitate cardiotoxicity assessment in drug development. Assay Drug Dev Technol. 2011 Dec;9(6):580-8. doi: 10.1089/adt.2011.0425.</a></p>

<p class="is-size-6"> <strong> Dataset License: </strong> Not Specified. <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>

<p class="is-size-6"> <strong> Contributed by: </strong> <a href="https://bbirnbaum.com">Ben Birnbaum</a>.</p>

<hr />

<h3 id="tox21">Tox21</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong> Tox21 is a data challenge which contains qualitative toxicity measurements for 7,831 compounds on 12 different targets, such as nuclear receptors and stree response pathways. </p>

<p class="is-size-6">  <strong> Task Description: </strong>Binary classification. Given a drug SMILES string, predict the toxicity in a specific assay. </p>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> Depends on various assays ~6,000 drugs. </p>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split</span> <span class="tag is-info is-light">Scaffold Split</span>  </p>

<p class="is-size-6">  <strong> Note: </strong> Tox21 contains multiple assays data. To retrieve the specific labels for that assay, specify the label name in the <code>label_name</code> variable to the data loader. You can find all available label names by calling: </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.utils</span> <span class="kn">import</span> <span class="n">retrieve_label_name_list</span>
<span class="n">label_list</span> <span class="o">=</span> <span class="n">retrieve_label_name_list</span><span class="p">(</span><span class="s">'Tox21'</span><span class="p">)</span>
</code></pre></div></div>

<p class="is-size-6"> Then, go to the standard TDC data loader procedure with the label name specified. </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">Tox</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">Tox</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'Tox21'</span><span class="p">,</span> <span class="n">label_name</span> <span class="o">=</span> <span class="n">label_list</span><span class="p">[</span><span class="mi">0</span><span class="p">])</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6">  <strong> References: </strong>  </p>

<p><a href="https://www.frontiersin.org/research-topics/2954/tox21-challenge-to-build-predictive-models-of-nuclear-receptor-and-stress-response-pathways-as-media">[1] Tox21 Challenge.</a></p>

<p class="is-size-6"> <strong> Dataset License: </strong> Not Specified. <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>

<hr />

<h3 id="toxcast">ToxCast</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong> ToxCast includes qualitative results of over 600 experiments on 8k compounds. </p>

<p class="is-size-6">  <strong> Task Description: </strong>Binary classification. Given a drug SMILES string, predict the toxicity in a specific assay. </p>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> Depends on various assays from couple hundres to thousands of drugs. </p>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split</span> <span class="tag is-info is-light">Scaffold Split</span>  </p>

<p class="is-size-6">  <strong> Note: </strong> ToxCast contains multiple assays data. To retrieve the specific labels for that assay, specify the label name in the <code>label_name</code> variable to the data loader. You can find all available label names by calling: </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.utils</span> <span class="kn">import</span> <span class="n">retrieve_label_name_list</span>
<span class="n">label_list</span> <span class="o">=</span> <span class="n">retrieve_label_name_list</span><span class="p">(</span><span class="s">'Toxcast'</span><span class="p">)</span>
</code></pre></div></div>

<p class="is-size-6"> Then, go to the standard TDC data loader procedure with the label name specified. </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">Tox</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">Tox</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'ToxCast'</span><span class="p">,</span> <span class="n">label_name</span> <span class="o">=</span> <span class="n">label_list</span><span class="p">[</span><span class="mi">0</span><span class="p">])</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6">  <strong> References: </strong>  </p>

<p><a href="https://pubmed.ncbi.nlm.nih.gov/27367298/">[1] Richard, Ann M., et al. “ToxCast chemical landscape: paving the road to 21st century toxicology.” Chemical research in toxicology 29.8 (2016): 1225-1251.</a></p>

<p class="is-size-6"> <strong> Dataset License: </strong> <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>

<hr />
{% endif %}

{% if task == "hts" %}

<h3 id="butkiewicz-et-al">Butkiewicz et al.</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong> These are nine high-quality high-throughput screening (HTS) datasets from [1]. These datasets were curated from HTS data at the PubChem database [2]. Typically, HTS categorizes small molecules into hit, inactive, or unspecified against a certain therapeutic target. However, a compound may be falsely classified as a hit due to experimental artifacts such as optical interference. Moreover, because the screening is performed without duplicates, and the cutoff is often set loose to minimize the false negative rates, the results from the primary screens often contain high false positive rates [3]. Hence the result from the primary screen is only used as the first iteration to reduce the compound library to a smaller set of further confirmatory tests.

Here each dataset is carefully collated through confirmation screens to validate active compounds. The curation process is documented in [1]. Each dataset is identified by the PubChem Assay ID (AID).

Features of the datasets: (1) At least 150 confirmed active compounds present; (2) Diverse target classes; (3) Realistic (large number and highly imbalanced label).
 </p>

<p class="is-size-6">  <strong> Task Description: </strong>Binary classification. Given a compound SMILES string, predict its activity against a diverse set of targets. </p>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> </p>

<table>
  <thead>
    <tr>
      <th>Protein Target Class</th>
      <th>PubChem AID</th>
      <th>Protein Target</th>
      <th>Total # of Molecules</th>
      <th># of Active Molecules</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>GPCR</td>
      <td>435008</td>
      <td>Orexin1 Receptor</td>
      <td>218,158</td>
      <td>233</td>
    </tr>
    <tr>
      <td>GPCR</td>
      <td>1798</td>
      <td>M1 Muscarinic Receptor Agonists</td>
      <td>61,833</td>
      <td>187</td>
    </tr>
    <tr>
      <td>GPCR</td>
      <td>435034</td>
      <td>M1 Muscarinic Receptor Antagonists</td>
      <td>61,756</td>
      <td>362</td>
    </tr>
    <tr>
      <td>Ion Channel</td>
      <td>1843</td>
      <td>Potassium Ion Channel Kir2.1</td>
      <td>301,493</td>
      <td>172</td>
    </tr>
    <tr>
      <td>Ion Channel</td>
      <td>2258</td>
      <td>KCNQ2 Potassium Channel</td>
      <td>302,405</td>
      <td>213</td>
    </tr>
    <tr>
      <td>Ion Channel</td>
      <td>463087</td>
      <td>Cav3 T-type Calcium Channels</td>
      <td>100,875</td>
      <td>703</td>
    </tr>
    <tr>
      <td>Transporter</td>
      <td>488997</td>
      <td>Choline Transporter</td>
      <td>302,306</td>
      <td>252</td>
    </tr>
    <tr>
      <td>Kinase</td>
      <td>2689</td>
      <td>Serine/Threonine Kinase 33</td>
      <td>319,792</td>
      <td>172</td>
    </tr>
    <tr>
      <td>Enzyme</td>
      <td>485290</td>
      <td>Tyrosyl-DNA Phosphodiesterase</td>
      <td>341,365</td>
      <td>281</td>
    </tr>
  </tbody>
</table>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split</span> <span class="tag is-info is-light">Scaffold Split</span>  </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">HTS</span>
<span class="c1"># Orexin1 Receptor
</span><span class="n">data</span> <span class="o">=</span> <span class="n">HTS</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'orexin1_receptor_butkiewicz'</span><span class="p">)</span>
<span class="n">df</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_data</span><span class="p">()</span>
<span class="n">splits</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>

<span class="c1"># M1 Muscarinic Receptor Agonists
</span><span class="n">data</span> <span class="o">=</span> <span class="n">HTS</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'m1_muscarinic_receptor_agonists_butkiewicz'</span><span class="p">)</span>
<span class="n">df</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_data</span><span class="p">()</span>
<span class="n">splits</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>

<span class="c1"># M1 Muscarinic Receptor Antagonists
</span><span class="n">data</span> <span class="o">=</span> <span class="n">HTS</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'m1_muscarinic_receptor_antagonists_butkiewicz'</span><span class="p">)</span>
<span class="n">df</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_data</span><span class="p">()</span>
<span class="n">splits</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>

<span class="c1"># Potassium Ion Channel Kir2.1 
</span><span class="n">data</span> <span class="o">=</span> <span class="n">HTS</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'potassium_ion_channel_kir2.1_butkiewicz'</span><span class="p">)</span>
<span class="n">df</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_data</span><span class="p">()</span>
<span class="n">splits</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>

<span class="c1"># KCNQ2 Potassium Channel
</span><span class="n">data</span> <span class="o">=</span> <span class="n">HTS</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'kcnq2_potassium_channel_butkiewicz'</span><span class="p">)</span>
<span class="n">df</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_data</span><span class="p">()</span>
<span class="n">splits</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>

<span class="c1"># Cav3 T-type Calcium Channels
</span><span class="n">data</span> <span class="o">=</span> <span class="n">HTS</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'cav3_t-type_calcium_channels_butkiewicz'</span><span class="p">)</span>
<span class="n">df</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_data</span><span class="p">()</span>
<span class="n">splits</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>

<span class="c1"># Choline Transporter
</span><span class="n">data</span> <span class="o">=</span> <span class="n">HTS</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'choline_transporter_butkiewicz'</span><span class="p">)</span>
<span class="n">df</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_data</span><span class="p">()</span>
<span class="n">splits</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>

<span class="c1"># Serine/Threonine Kinase 33
</span><span class="n">data</span> <span class="o">=</span> <span class="n">HTS</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'serine_threonine_kinase_33_butkiewicz'</span><span class="p">)</span>
<span class="n">df</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_data</span><span class="p">()</span>
<span class="n">splits</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>

<span class="c1"># Tyrosyl-DNA Phosphodiesterase
</span><span class="n">data</span> <span class="o">=</span> <span class="n">HTS</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'tyrosyl-dna_phosphodiesterase_butkiewicz'</span><span class="p">)</span>
<span class="n">df</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_data</span><span class="p">()</span>
<span class="n">splits</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6">  <strong> References: </strong>  </p>

<p><a href="https://www.mdpi.com/1420-3049/18/1/735">[1] Butkiewicz, Mariusz, et al. “Benchmarking ligand-based virtual High-Throughput Screening with the PubChem database.” Molecules 18.1 (2013): 735-756.</a></p>

<p><a href="https://academic.oup.com/nar/article/47/D1/D1102/5146201">[2] Kim, Sunghwan, et al. “PubChem 2019 update: improved access to chemical data.” Nucleic acids research 47.D1 (2019): D1102-D1109.</a></p>

<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5962024/">[3] Butkiewicz, Mariusz, et al. “High-throughput screening assay datasets from the pubchem database.” Chemical informatics (Wilmington, Del.) 3.1 (2017).</a></p>

<p class="is-size-6"> <strong> Dataset License: </strong> Not Specified. <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>

<hr />

{% endif %}

{% if task == "qm" %}
<h3 id="qm7b">QM7b</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong> QM7 is a subset of GDB-13 (a database of nearly 1 billion stable and synthetically accessible organic molecules) composed of all molecules of up to 23 atoms (including 7 heavy atoms C, N, O, and S), where 14 properties (e.g. polarizability, HOMO and LUMO eigenvalues, excitation energies) have to be predicted at different levels of theory (ZINDO, SCS, PBE0, GW). </p>

<p class="is-size-6">  <strong> Task Description: </strong>Regression. Given a drug 3D xyz coordinates, predict the drug property. </p>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> 7,211 drugs. </p>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split</span> </p>

<p class="is-size-6">  <strong> Note: </strong> QM7b contains multiple properties. To retrieve the specific labels for that property, specify the property name in the <code>label_name</code> variable to the data loader. You can find all available label names by calling: </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.utils</span> <span class="kn">import</span> <span class="n">retrieve_label_name_list</span>
<span class="n">label_list</span> <span class="o">=</span> <span class="n">retrieve_label_name_list</span><span class="p">(</span><span class="s">'QM7b'</span><span class="p">)</span>
</code></pre></div></div>

<p class="is-size-6"> Then, go to the standard TDC data loader procedure with the label name specified. </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">QM</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">QM</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'QM7b'</span><span class="p">,</span> <span class="n">label_name</span> <span class="o">=</span> <span class="n">label_list</span><span class="p">[</span><span class="mi">0</span><span class="p">])</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6">  <strong> References: </strong>  </p>

<p><a href="https://pubs.acs.org/doi/abs/10.1021/ja902302h">[1] Blum, Lorenz C., and Jean-Louis Reymond. “970 million druglike small molecules for virtual screening in the chemical universe database GDB-13.” Journal of the American Chemical Society 131.25 (2009): 8732-8733.</a></p>

<p><a href="https://iopscience.iop.org/article/10.1088/1367-2630/15/9/095003/meta">[2] Montavon, Grégoire, et al. “Machine learning of molecular electronic properties in chemical compound space.” New Journal of Physics 15.9 (2013): 095003.</a></p>

<p class="is-size-6"> <strong> Dataset License: </strong> Not Specified. <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>

<hr />

<h3 id="qm8">QM8</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong> Electronic spectra and excited state energy of small molecules calculated by multiple quantum mechanic methods. Consisting of low-lying singlet-singlet vertical electronic spectra of over 20 000 synthetically feasible small organic molecules with up to eight CONF atom. From MoleculeNet and loaded from DeepChem. </p>

<p class="is-size-6">  <strong> Task Description: </strong>Regression. Given a drug 3D xyz coordinates, predict the drug property. </p>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> 21,786 drugs. </p>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split</span> </p>

<p class="is-size-6">  <strong> Note: </strong> QM8 contains multiple properties. To retrieve the specific labels for that property, specify the property name in the <code>label_name</code> variable to the data loader. You can find all available label names by calling: </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.utils</span> <span class="kn">import</span> <span class="n">retrieve_label_name_list</span>
<span class="n">label_list</span> <span class="o">=</span> <span class="n">retrieve_label_name_list</span><span class="p">(</span><span class="s">'QM8'</span><span class="p">)</span>
</code></pre></div></div>

<p class="is-size-6"> Then, go to the standard TDC data loader procedure with the label name specified. </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">QM</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">QM</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'QM8'</span><span class="p">,</span> <span class="n">label_name</span> <span class="o">=</span> <span class="n">label_list</span><span class="p">[</span><span class="mi">0</span><span class="p">])</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6">  <strong> References: </strong>  </p>

<p><a href="https://pubs.acs.org/doi/abs/10.1021/ci300415d">[1] Ruddigkeit, Lars, et al. “Enumeration of 166 billion organic small molecules in the chemical universe database GDB-17.” Journal of chemical information and modeling 52.11 (2012): 2864-2875.</a></p>

<p><a href="https://aip.scitation.org/doi/abs/10.1063/1.4928757?casa_token=mVBximpm4AMAAAAA:iG5mbej2eVDthAfXQFx7LIcQ4-b3I9W_5NsUxE_3LjDRHgGzgqJe-LfCmBiPy1W-ZhjO3oB0RXyW">[2] Ramakrishnan, Raghunathan, et al. “Electronic spectra from TDDFT and machine learning in chemical space.” The Journal of chemical physics 143.8 (2015): 084111.</a></p>

<p><a href="https://books.google.com/books?hl=en&amp;lr=&amp;id=6OiRDwAAQBAJ&amp;oi=fnd&amp;pg=PT19&amp;dq=deep+learning+for+life+science+Bharath+Ramsundar&amp;ots=SwHP1HB_qK&amp;sig=iox0o4HN7pBkWC-Bw0X1qDNfwwQ">[3] Ramsundar, Bharath, et al. Deep learning for the life sciences: applying deep learning to genomics, microscopy, drug discovery, and more. “ O’Reilly Media, Inc.”, 2019.</a></p>

<p><a href="https://pubs.rsc.org/--/content/articlehtml/2018/sc/c7sc02664a">[4] Wu, Zhenqin, et al. “MoleculeNet: a benchmark for molecular machine learning.” Chemical science 9.2 (2018): 513-530.</a></p>

<p class="is-size-6"> <strong> Dataset License: </strong> <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>

<hr />

<h3 id="qm9">QM9</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong> computed geometric, energetic, electronic, and thermodynamic properties for 134k stable small organic molecules made up of CHONF. These molecules correspond to the subset of all 133,885 species with up to nine heavy atoms (CONF) out of the GDB-17 chemical universe of 166 billion organic molecules. We report geometries minimal in energy, corresponding harmonic frequencies, dipole moments, polarizabilities, along with energies, enthalpies, and free energies of atomization. All properties were calculated at the B3LYP/6-31G(2df,p) level of quantum chemistry. From MoleculeNet and loaded from DeepChem. </p>

<p class="is-size-6">  <strong> Task Description: </strong>Regression. Given a drug 3D xyz coordinates, predict the drug property. </p>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> 133,885 drugs. </p>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split</span> </p>

<p class="is-size-6">  <strong> Note: </strong> QM8 contains multiple properties. To retrieve the specific labels for that property, specify the property name in the <code>label_name</code> variable to the data loader. You can find all available label names by calling: </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.utils</span> <span class="kn">import</span> <span class="n">retrieve_label_name_list</span>
<span class="n">label_list</span> <span class="o">=</span> <span class="n">retrieve_label_name_list</span><span class="p">(</span><span class="s">'QM9'</span><span class="p">)</span>
</code></pre></div></div>

<p class="is-size-6"> Then, go to the standard TDC data loader procedure with the label name specified. </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">QM</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">QM</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'QM9'</span><span class="p">,</span> <span class="n">label_name</span> <span class="o">=</span> <span class="n">label_list</span><span class="p">[</span><span class="mi">0</span><span class="p">])</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6">  <strong> References: </strong>  </p>

<p><a href="https://pubs.acs.org/doi/abs/10.1021/ci300415d">[1] Ruddigkeit, Lars, et al. “Enumeration of 166 billion organic small molecules in the chemical universe database GDB-17.” Journal of chemical information and modeling 52.11 (2012): 2864-2875.</a></p>

<p><a href="https://aip.scitation.org/doi/abs/10.1063/1.4928757?casa_token=mVBximpm4AMAAAAA:iG5mbej2eVDthAfXQFx7LIcQ4-b3I9W_5NsUxE_3LjDRHgGzgqJe-LfCmBiPy1W-ZhjO3oB0RXyW">[2] Ramakrishnan, Raghunathan, et al. “Electronic spectra from TDDFT and machine learning in chemical space.” The Journal of chemical physics 143.8 (2015): 084111.</a></p>

<p><a href="https://books.google.com/books?hl=en&amp;lr=&amp;id=6OiRDwAAQBAJ&amp;oi=fnd&amp;pg=PT19&amp;dq=deep+learning+for+life+science+Bharath+Ramsundar&amp;ots=SwHP1HB_qK&amp;sig=iox0o4HN7pBkWC-Bw0X1qDNfwwQ">[3] Ramsundar, Bharath, et al. Deep learning for the life sciences: applying deep learning to genomics, microscopy, drug discovery, and more. “ O’Reilly Media, Inc.”, 2019.</a></p>

<p><a href="https://pubs.rsc.org/--/content/articlehtml/2018/sc/c7sc02664a">[4] Wu, Zhenqin, et al. “MoleculeNet: a benchmark for molecular machine learning.” Chemical science 9.2 (2018): 513-530.</a></p>

<p class="is-size-6"> <strong> Dataset License: </strong> <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>

<hr />
{% endif %}

{% if task == "develop" %}
<h3 id="tap">TAP</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong> Immunogenicity, instability, self-association, high viscosity, polyspecificity, or poor expression can all preclude an antibody from becoming a therapeutic. Early identification of these negative characteristics is essential. Akin to the Lipinski guidelines, which measure druglikeness in small molecules, Therapeutic Antibody Profiler (TAP) highlights antibodies that possess characteristics that are rare/unseen in clinical-stage mAb therapeutics. In this dataset, TDC includes five metrics measuring developability of an antibody: CDR length, patches of surface hydrophobicity (PSH), patches of positive charge (PPC), patches of negative charge (PNC), structural Fv charge symmetry parameter (SFvCSP). </p>

<p class="is-size-6">  <strong> Task Description: </strong>Regression. Given the antibody's heavy chain and light chain sequence, predict its developability. The input X is a list of two sequences where the first is the heavy chain and the second light chain.</p>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> 242 antibodies. </p>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split</span> </p>

<p class="is-size-6">  <strong> Note: </strong> TAP contains five developability metrics. To retrieve the specific labels for that metric, specify the label name in the <code>label_name</code> variable to the data loader. You can find all available label names by calling: </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.utils</span> <span class="kn">import</span> <span class="n">retrieve_label_name_list</span>
<span class="n">label_list</span> <span class="o">=</span> <span class="n">retrieve_label_name_list</span><span class="p">(</span><span class="s">'TAP'</span><span class="p">)</span>
</code></pre></div></div>

<p class="is-size-6"> Then, go to the standard TDC data loader procedure with the label name specified. </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">Develop</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">Develop</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'TAP'</span><span class="p">,</span> <span class="n">label_name</span> <span class="o">=</span> <span class="n">label_list</span><span class="p">[</span><span class="mi">0</span><span class="p">])</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6">  <strong> References: </strong>  </p>

<p><a href="https://www.pnas.org/content/116/10/4025.short">[1] Raybould, Matthew IJ, et al. “Five computational developability guidelines for therapeutic antibody profiling.” Proceedings of the National Academy of Sciences 116.10 (2019): 4025-4030.</a></p>

<p class="is-size-6"> <strong> Dataset License: </strong>  <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>

<hr />

<h3 id="sabdab-chen-et-al">SAbDab, Chen et al.</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong> Antibody data from Chen et al, where they process from the SAbDab. From an initial dataset of 3816 antibodies, they retained 2426 antibodies that satisfy the following criteria: 1. have both sequence (FASTA) and Protein Data Bank (PDB) structure files, 2. contain both a heavy chain and a light chain, and 3. have crystal structures with resolution &lt; 3 Å. The DI label is derived from BIOVIA's pipelines.  </p>

<p class="is-size-6">  <strong> Task Description: </strong>Binary classification. Given the antibody's heavy chain and light chain sequence, predict its developability. The input X is a list of two sequences where the first is the heavy chain and the second light chain.</p>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> 2,409 antibodies. </p>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split</span> </p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">Develop</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">Develop</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'SAbDab_Chen'</span><span class="p">)</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6">  <strong> Note: </strong> Since <code>0.3.7</code>, TDC now supports graphein protein representation for this task. To obtain protein representation, see a tutorial <a href="https://github.com/mims-harvard/TDC/blob/main/tutorials/graphein_demo_developability.ipynb"> here! </a> </p>

<p class="is-size-6">  <strong> References: </strong>  </p>

<p><a href="https://www.biorxiv.org/content/10.1101/2020.06.18.159798v1.abstract">[1] Chen, Xingyao, et al. “Predicting antibody developability from sequence using machine learning.” bioRxiv (2020).
</a></p>

<p><a href="https://academic.oup.com/nar/article-abstract/42/D1/D1140/1044118">[2] Dunbar, James, et al. “SAbDab: the structural antibody database.” Nucleic acids research 42.D1 (2014): D1140-D1146.
</a></p>

<p><a href="">[3] Biovia, Dassault Systèmes. “BIOVIA pipeline pilot.” Dassault Systèmes: San Diego, BW, Release (2017).
</a></p>

<p class="is-size-6"> <strong> Dataset License: </strong>  <a href="https://creativecommons.org/licenses/by/3.0/">CC BY 3.0</a>.</p>

<hr />

{% endif %}

{% if task == "CRISPROutcome" %}
<h3 id="leenay-et-al">Leenay et al.</h3>

<p class="is-size-6">  <strong> Dataset Description: </strong> Primary T cells are a promising cell type for therapeutic genome editing, as they can be engineered efficiently ex vivo and transferred to patients. This dataset consists of the DNA repair outcomes of CRISPR-CAS9 knockout experiments on primary CD4+ T cells drawn from 15 donors. For each of the 1,521 unique genomic locations from 553 genes, we provide the 20-nucleotide guide sequence along with the 3-nucletoide PAM sequence. </p>

<p class="is-size-6"> 5 repair outcomes are included for prediction: fraction of indel reads with an insertion, average insertion length, average deletion length, indel diversity, fraction of repair outcomes with a frameshift. </p>

<p class="is-size-6">  <strong> Task Description: </strong>Regression. Given a DNA sequence, predict the repair outcomes of a CRISPR-CAS9 knockout experiment. </p>

<p class="is-size-6">  <strong> Dataset Statistics: </strong> 1,521 DNA sequences (guide+PAM) with 5 repair outcomes for each measured across different donor populations of primary T cells. </p>

<p class="is-size-6">  <strong> Dataset Split: </strong> <span class="tag is-info is-light">Random Split</span> </p>

<p>To obtain the label names,</p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.utils</span> <span class="kn">import</span> <span class="n">retrieve_label_name_list</span>
<span class="n">label_list</span> <span class="o">=</span> <span class="n">retrieve_label_name_list</span><span class="p">(</span><span class="s">'Leenay'</span><span class="p">)</span>
</code></pre></div></div>

<p>Then, proceed with the usual data loader:</p>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">tdc.single_pred</span> <span class="kn">import</span> <span class="n">CRISPROutcome</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">CRISPROutcome</span><span class="p">(</span><span class="n">name</span> <span class="o">=</span> <span class="s">'Leenay'</span><span class="p">,</span> <span class="n">label_name</span> <span class="o">=</span> <span class="n">label_list</span><span class="p">[</span><span class="mi">0</span><span class="p">])</span>
<span class="n">split</span> <span class="o">=</span> <span class="n">data</span><span class="p">.</span><span class="n">get_split</span><span class="p">()</span>
</code></pre></div></div>

<p class="is-size-6">  <strong> References: </strong>  </p>

<p><a href="https://idp.nature.com/authorize/casa?redirect_uri=https://www.nature.com/articles/s41587-019-0203-2&amp;casa_token=nIKOlfHR0fkAAAAA:LspabShsQSiLsP0z4vu0xYJWwIOWLbWfKBTwIwNgEFNuSan3vxMKI133KNyMTB6onOYHAS7Ql27lkl4_Og">[1] Leenay, Ryan T., et al. “Large dataset enables prediction of repair after CRISPR–Cas9 editing in primary T cells.” Nature biotechnology 37.9 (2019): 1034-1037.</a></p>

<p class="is-size-6"> <strong> Dataset License: </strong><a href="https://creativecommons.org/licenses/by/3.0/">CC BY 3.0</a>.</p>

<hr />
{% endif %}


</div>
                </div>
                
            </div>
        </div>
    </section>
    
        <footer class="footer">
    <div class="container">
        
        <div class="columns is-mobile">
            <div class="column is-8 has-text-left is-vcentered">
                <a class="navbar-brand" href="/">
                    <span><img src="{{ url_for('static', filename='images/tdc_horizontal.png') }}" alt="Logo" style="max-height: 40px; max-width: 250px;"></span>
                </a>
            </div>
            <div class="column is-4 has-text-right is-vcentered">
                <a href="https://zitniklab.hms.harvard.edu/publications/papers/TDC-neurips21-main.pdf">
                    <span class="icon is-large">
                      <i class="fas fa-file-alt fa-3x"></i>
                    </span>
                </a>

                <a href="https://github.com/mims-harvard/TDC">
                    <span class="icon is-large">
                      <i class="fas fab fa-github fa-3x"></i>
                    </span>
                </a>

                <a href="https://twitter.com/ProjectTDC">
                    <span class="icon is-large">
                      <i class="fas fab fa-twitter fa-3x"></i>
                    </span>
                </a>

                <a href="https://join.slack.com/t/pytdc/shared_invite/zt-x0ujg5v6-zwtQZt83fhRdgrYjXRFz5g">
                    <span class="icon is-large">
                      <i class="fas fab fa-slack fa-3x"></i>
                    </span>
                </a>
            </div>
        </div>
        
    </div>
</footer>
    
    <script src="/assets/js/app.js" type="text/javascript"></script><!-- footer scripts --></body>
</html>

